plugins/web/opentelemetry-plugin-user-interaction/src/userInteraction.ts

```diff
@@ -227,6 +227,13 @@ export class UserInteractionPlugin extends BasePlugin<unknown> {
     return patched;
   }
 
+  // utility method to deal with the Function|EventListener nature of addEventListener
+  private _invokeListener(listener: any, target: any, args: any[]): any {
```

Comment:
Use of any type here for listener is not ideal. If you really don't know what will be sent in, an unknown would be better which would require you to check instanceof EventEmitter or typeof === 'function' and prevent errrors.

Reply:
I wanted to type it as `Function|EventListener` but found compilation problems; I just now discovered that what I meant was `Function|EventListenerObject` so I've changed all the types to that instead.  Better?